src/backend/marsha/core/views.py

```diff
@@ -131,19 +155,31 @@ def post(self, request, *args, **kwargs):
         return self.render_to_response(self.get_context_data())
 
 
-@method_decorator(csrf_exempt, name="dispatch")
-@method_decorator(xframe_options_exempt, name="dispatch")
-class OpenEdxVideoLTIView(VideoLTIView):
-    """View called by the deprecated LTI launch request.
+class VideoLTIView(BaseLTIView):
+    """Video view called by an LTI launch request."""
 
-    [DEPRECATED] It is kept here for backward compatibility with the first versions of Marsha,
-    only works with Open edX and will be removed in future releases.
+    model = Video
+    serializer_class = VideoSerializer
 
-    """
+    def _get_or_create_resource(self, lti):
```

Comment:
you don't need this anymore if you call in the BaseFile class you call:
self.get_or_create_resource(self.model, lti)

Reply:
:+1: